src/emitters/templates/py_interface.jinja2

```diff
@@ -232,18 +232,10 @@ class XBVCEdgePoint(object):
 
     def stop(self):
         self._stop_event.set()
-        try:
-            self._recv_thread_handle.join()
-        except:
-            pass
-        
+
+        self._recv_thread_handle.join()
```

Comment:
why remove the exception catching?

Reply:
Actually not 100% sure, this is a change that I pulled down from original repo.  I can add it back in tonight.

Reply:
Yes, let's keep the logic exactly as we have it in our forked repo - the RC app and upstream code may depend on this behavior. Thanks!

Reply:
Fixed